src/Chapter4.hs

```diff
@@ -611,6 +622,18 @@ Implement the 'Monad' instance for our lists.
   maybe a few) to flatten lists of lists to a single list.
 -}
 
+-- I couldn't complete this.
+-- I looked at the solutions and it wouldn't compile either.
+
+{-
+instance Monad List where
+  (>>=) :: List a -> (a -> List b) -> List b
+  list >>= f = flatHelper (fmap f l)
+
+flatHelper :: List (List a) -> List a
+flatHelper Empty = Empty
+flatHelper (Cons x xs) =  x : (flatHelper xs)
```

Comment:
Here `x` is list itself as this is the list of lists inside 🙂 
So `:` won't work, you need to use some `append` function to get 2 lists together 🙂